docs/csharp/quick-starts/branches-and-loops.yml

```diff
@@ -45,6 +45,9 @@ items:
 
     This process of checking conditions and executing statements based on those conditions is very powerful. Let's explore more.
 
+    > [!NOTE]
+    > This online coding experience is in preview mode. It's been working well, but if you encounter problems, please report them [here on our GitHub repo](https://github.com/dotnet/try/issues).
```

Comment:
I'd probably remove this part "It's been working well, but". I don't think it's needed.

And for links, we usually avoid using words like here. So how about instead of [here on our GitHub repo] we have 
"on the [dotnet/try repo]."